terraform/digitalocean/k8s/variables.tf

```diff
@@ -14,7 +14,7 @@ variable "region" {
   description = "(Required) Provide DigitalOcean region"
 }
 
-variable "version" {
+variable "k8s_version" {
   type        = string
   description = "Provide DigitalOcean Kubernetes minor version (e.g. '1.16' or '1.15')"
   default     = "1.16"
```

Comment:
If you'd like and @voatsap accept, we can add experimental check functional that variable is 1.15 or 1.16
https://www.terraform.io/docs/configuration/variables.html#custom-validation-rules

Reply:
Yes looks smart, but we need to align this every time with docs/OPTIONS.md

Reply:
JFYI, it will be stable in TF 0.13.0.

By the way, we can just check only format like

```terraform
variable "k8s_version" {
  type        = string
  description = "Provide DigitalOcean Kubernetes minor version (e.g. '1.16' or '1.15')"
  default     = "1.17"

  validation {
    # regex(...) fails if it cannot find a match
    condition     = can(regex("^[1-9]\.[0-9]{1,2}$", var.k8s_version))
    error_message = "The k8s_version value must be a valid K8s version."
  }
```

Reply:
Lets don't make this strict check that time. Its could be changed in provider side any time, so we don't need to change these checks every time